kp.go

```diff
@@ -253,6 +253,9 @@ func (c *Client) do(ctx context.Context, req *http.Request, res interface{}) (*h
 
 	switch response.StatusCode {
 	case http.StatusOK, http.StatusCreated:
+		if req.Method == http.MethodHead {
```

Comment:
Does `HEAD` return 200 OK or No Content...?

Reply:
it is returning 200

Reply:
Yep... weird.